quickstart.schema.json

```diff
@@ -162,6 +162,21 @@
             }
           },
           "additionalProperties": true
+        },
+        "labels": {
+          "$id": "#/properties/metadata/properties/labels",
+          "type": "object",
+          "examples": [
+            {
+              "labels": {
+                "service-category/application-services": true,
+                "product/rhosak": true,
+                "product/rhosr": true,
```

Comment:
If this file is used to define the various properties, should we include an entry for `product/rhoad` and `product/rhoc` here too? And `"technology":  "cli"` too?

We also seem to be using additional double quotation marks in the `product` and `technology` entries in this file, but maybe the syntax is different in the schema file?

Reply:
Thanks, Breda - my understanding is that the entries are all only "Examples" and are not used. I asked that question about adding extra lines for other products and technology but Juntao confirmed it is not necessary as they are just examples. The double quotations are necessary for a schema json file.